app/assets/javascripts/browser-update.js

```diff
@@ -33,7 +33,7 @@ var $buo = function() {
   }
 
   // retrieve localized browser upgrade text
-  var t = <%= "'" + I18n.t('js.browser_update') + "'" %>;
+  var t = I18n.translate('js.browser_update');
```

Comment:
Did you test this change?

When the code in this file is run, there is no i18n global, since the application did not load due to the incompatible environment.

Reply:
Yup, i ran it on localhost and window.I18n was present and proper string was inserted.
It is also true for `meta.discourse.org`.

<img width="581" alt="image" src="https://user-images.githubusercontent.com/546845/99569427-e7926700-29d0-11eb-8b2c-73a14e90d68f.png">